src/utils/operations/tests/getDenominatedValue.test.ts

```diff
@@ -0,0 +1,68 @@
+import { baseTransactionMock } from 'components/TransactionsInterpreter/helpers/tests/base-transaction-mock';
```

Comment:
It seems `getDenominatedValue.ts` is newly added. Better change name directly to `getFormattedAmount.ts`